lib/mauth/client.rb

```diff
@@ -299,13 +304,17 @@ def authenticate!(object)
         time_valid!(object)
         token_valid!(object)
         signature_valid!(object)
-      rescue InauthenticError
+      rescue MauthNotPresent => e
+        logger.warn "mAuth signature authentication failed for #{object.class}."
+        logger.warn e.message
+        raise
+      rescue InauthenticError => e
         logger.error "mAuth signature authentication failed for #{object.class}. encountered error:"
```

Comment:
There are hundreds of these "errors" in production per day that occur on a regular schedule (for the most part) for each service, so it seems like this is also a candidate for using the `warn` level.

Reply:
I do not care one way or the other but I think those errors come from the above case (no headers available) which previously would be rescued here.
Now this case catches time issues and "token issues" which I'm unsure what those are.

Do you still want to convert it to warn?

Reply:
Ah, duh -- leave it as `error`.  Thanks!